lzy/iam/src/main/java/ai/lzy/iam/storage/impl/DbAuthService.java

```diff
@@ -81,15 +84,15 @@ private Subject authenticateOtt(OttCredentials ott) {
                 Subject result = null;
                 if (expiredAt.isAfter(Instant.now())) {
                     result = switch (subjectType) {
-                        case USER -> new User(subjectId);
-                        case WORKER -> new Worker(subjectId);
+                        case USER -> throw new AuthInternalException("USER subject is incompatible with OTT auth");
+                        case WORKER -> throw new AuthInternalException("WORKER subject is incompatible with JWT auth");
```

Comment:
Why JWT? Could be a typo. We are using OTT here.

Reply:
will be fixed in the next PR